### PR TITLE
Added some info in Footer

### PR DESCRIPTION
**Pull request by simplews**
Added Short description with copyright and sitemap on Footer (see screenshot)
screen shot 2017-02-13 at 13 30 13

### CODE REVIEW

File: app/views/layouts/_footer.html.slim [hunk 1]
```diff
@@ -1,7 +1,44 @@
...
+                  | Карта звернень
+              li
+                = link_to 'Стрічка оновлень', feed_path
+              li
+                = link_to 'Відкриті звернення', issues_path(status: 'opened')
+              li
+                = link_to 'Закриті звернення', issues_path(status: 'closed')
+          li
+            | Інформація
+            b.caret
```
Comment (igorkasyanchuk) [inline app/views/layouts/_footer.html.slim]: remove  b.caret from footer

Comment (simplews) [inline app/views/layouts/_footer.html.slim]: Removed

File: app/views/layouts/application.html.slim [hunk 2]
```diff
@@ -9,5 +9,5 @@ html
         = yield
         = any_login_here if defined?(AnyLogin)
       #footer
-    = render 'layouts/footer'
+      = render 'layouts/footer'
```
Comment (skcc321) [inline app/views/layouts/application.html.slim]: probably should be wrapped by #footer ?

Comment (simplews) [inline app/views/layouts/application.html.slim]: No, this layout uses sticky footer from Compass library, so we need use this markup: http://compass-style.org/reference/compass/layout/sticky_footer/

**Pull request merged**